packages/magento-customer/components/EditAddressForm/EditAddressForm.tsx

```diff
@@ -50,7 +50,7 @@ export function EditAddressForm(props: EditAddressFormProps) {
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
         houseNumber: address?.street?.[1],
```

Comment:
This can also be empty in theory. While you're at it, can you try and remove the ts-ignore?

Reply:
I'm not completely sure, but if it is currently working I wouldn't change it.